ee/clickhouse/models/event.py

```diff
@@ -19,7 +21,7 @@
 def create_event(
     event_uuid: uuid.UUID,
     event: str,
-    team: Team,
+    team: Union[Team, int],
```

Comment:
Don't think it makes sense to pollute the type - why can't we send the `Team` here?

Reply:
Yeah, we can actually accept `Team` only, no problem.